codalab/lib/zip_util.py

```diff
@@ -112,8 +112,7 @@ def pack_files_for_upload(
         'fileobj': <file object of archive>,
         'filename': <name of archive file>,
         'filesize': <size of archive in bytes, or None if unknown>,
-        'should_unpack': <True iff archive should be unpacked at server>,
-        'should_simplify': <True iff directory should be 'simplified' at server>
+        'should_unpack': <True iff archive should be unpacked at server>
```

Comment:
```suggestion
        'should_unpack': <True if the archive should be unpacked at server>
```

Reply:
I think this was intentional ("if and only if") and is used in other parts of the codebase, but we could probably think about style maybe in a separate issue.

Reply:
oh I wasn' t aware that `iff` was `if and only if`. It's fine then.